docs/v2/4_erc20-interactions.md

```diff
@@ -107,7 +113,7 @@ public class ERC20NameExample : MonoBehaviour
 
 ### Symbol {#symbol}
 
-Returns the symbol of the token, e.g. "USDC".
+Returns the symbol of an ERC-20 token associated with a specified contract address, e.g. "USDC".
```

Comment:
Use the same format for presenting. 

(e.g. "USDC")

Reply:
fixed